### PR TITLE
FS-1427; Refactor sandbox to use anchors, and get it ready to refactor each service

### PR DESCRIPTION
**Pull request by jakeschuurmans**
Use anchors to re-use data in sandbox values.yaml

prepare services helmchart templates to be moved to their metal-toolbox/<service> github repos

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -1,101 +1,137 @@
...
+      otel:
+        authenticate: false
+        url: jaeger:4317
+      fleetdb:
+        authenticate: false
+        url: http://fleetdb:8000
+        oidc_audience_url: # to be filled once oauth is available in the sandbox
+        oidc_issuer_url:   # to be filled once oauth is available in the sandbox
+        oidc_client_id: # to be filled by each service bellow
+        oidc_scopes:    # to be filled by each service bellow
```
Comment (DoctorVin) [inline values.yaml:27]: You might want to refer to the stuff you have below to save people making the same mistake I did. 😀

**Review by DoctorVin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Note: this assumes the KIND environment on your machine can connect to server BM
       --bmc-addr 192.168.1.1 \
       --bmc-user root \
       --bmc-pass hunter2 \
-      --server ede81024-f62a-4288-8730-3fab8cceabcc
+      --server ede81024-f62a-4288-8730-3fab8cceabcc\
```
[the remark this answers is not in the document]
Comment (jakeschuurmans) [reply README.md:62]: will do

File: templates/alloy-deployment.yaml [hunk 3]
```diff
@@ -36,33 +37,33 @@ spec:
               readOnly: true
           env:
             - name: ALLOY_COMPONENT_INVENTORY_ENDPOINT
-              value: "{{ .Values.alloy.env.ALLOY_COMPONENT_INVENTORY_ENDPOINT }}"
+              value: "{{ .Values.alloy.env.endpoints.component_inventory.url }}"
             - name: ALLOY_COMPONENT_INVENTORY_DISABLE_OAUTH
-              value: "{{ .Values.alloy.env.ALLOY_COMPONENT_INVENTORY_DISABLE_OAUTH }}"
+              value: "{{ not .Values.alloy.env.endpoints.component_inventory.authenticate }}"
             - name: ALLOY_FLEETDB_ENDPOINT
```
[the remark this answers is not in the document]
Comment (DoctorVin) [reply templates/alloy-deployment.yaml:43]: I wasn't clear. The token in `values.yaml` can have `fleetdb` in it, but the name of the environment variable must be `ALLOY_SERVERSERVICE_ENDPOINT`, cf. [this](https://github.com/metal-toolbox/alloy/blob/26374d074ae16435f7768d70743f505bca94da63/internal/app/config.go#L279)

**Review by DoctorVin — CHANGES_REQUESTED** [review]
Sorry, I was a dope and commented in `internal monologue` instead of English.

File: templates/alloy-deployment.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Alva8756) [reply templates/alloy-deployment.yaml]: It looks the main is using `fleetdb`?
https://github.com/metal-toolbox/alloy/blob/main/internal/app/config.go#L278-L280

The link using `serverservice` is from the comit: 26374d074ae16435f7768d70743f505bca94da63

[the remark this answers is not in the document]
Comment (DoctorVin) [reply templates/alloy-deployment.yaml]: My comment is from before Joel fixed Alloy 😁

**Pull request merged**